concepts/structs/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Structs allow to group pieces of related data together.",
```

Comment:
Slightly better grammar / simplification:

Structs group pieces of related data together.